exercises/static/exercises/vacuum_cleaner_loc_newmanager/python_template/ros2_humble/GUI.py

```diff
@@ -48,16 +49,8 @@ def __init__(self, host):
 
         self.shared_image = SharedImage("guiimage")
```

Comment:
This can also be removed...

Reply:
something to take a look at a later point...

Reply:
Removed in #2554